src/org/zaproxy/zap/extension/jxbrowserlinux64/selenium/JxBrowserProvider.java

```diff
@@ -99,8 +99,8 @@ private RemoteWebDriver getRemoteWebDriver(final String proxyAddress, final int
 
     private RemoteWebDriver getRemoteWebDriverImpl(String proxyAddress, int proxyPort) {
         try {
-            if (zbf == null) {
-                zbf = new ZapBrowserFrame(false, true, false, false);
+            if (zbf == null || zbf.isClosed()) {
```

Comment:
Should use different dialogues per `requesterId`(?), the AJAX Spider will interfere with the launched browsers (and the other way around).